pkg/controller/injector/k8s_binding_injector_test.go

```diff
@@ -29,6 +30,123 @@ import (
 	kapi "k8s.io/kubernetes/pkg/api"
 )
 
+func TestCreateSerializedSecret(t *testing.T) {
+	cases := []struct {
+		name   string
+		cred   *brokerapi.Credential
+		secret *v1.Secret
+	}{
+		{
+			name: "string type",
+			cred: &brokerapi.Credential{
+				"Hostname": "host",
+				"Port":     "123",
+				"Username": "user",
+				"Password": "password!@#!@#!0)",
+			},
+			secret: &v1.Secret{
+				Data: map[string][]byte{
+					"Hostname": []byte("host"),
+					"Port":     []byte("123"),
+					"Username": []byte("user"),
+					"Password": []byte("password!@#!@#!0)"),
+				},
+			},
+		},
+		{
+			name: "float type",
+			cred: &brokerapi.Credential{
+				"Hostname": "host",
+				"Port":     "123",
+				"Username": "user",
+				"Password": 1.23,
+			},
+			secret: &v1.Secret{
+				Data: map[string][]byte{
+					"Hostname": []byte("host"),
+					"Port":     []byte("123"),
+					"Username": []byte("user"),
+					"Password": []byte("1.23"),
```

Comment:
There is business logic in this code that needs to be tested.  Right now the business logic is simple - it's 'put everything in this deserialized response into a secret'.  Business logic is going to accumulate in this code, however - we're going to eventually be putting different pieces of information into different resources.  That is why we should test this code.